patients/add_edit_event_user.php

```diff
@@ -925,7 +925,7 @@ function deleteEvent() {
     return false;
  }
 
-    $(document).ready(function() {
+    $(document).ready(() => {
```

Comment:
neat, is this a shorthand for `function` in jquery?

Reply:
not jquery, its vanilla JS, just short hand for function.